ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/core/StandardComponentInitializer.java

```diff
@@ -121,6 +121,25 @@
 @Singleton
 public class StandardComponentInitializer {
 
+    public static final String NAVIGATE_TO_FILE    = "navigateToFile";
+    public static final String FULL_TEXT_SEARCH    = "fullTextSearch";
+    public static final String FIND_ACTION         = "findActionAction";
+    public static final String FORMAT              = "format";
+    public static final String COPY                = "copy";
+    public static final String CUT                 = "cut";
+    public static final String PASTE               = "paste";
+    public static final String SWITCH_LEFT_TAB     = "switchLeftTab";
+    public static final String SWITCH_RIGHT_TAB    = "switchRightTab";
+    public static final String OPEN_RECENT_FILES   = "openRecentFiles";
+    public static final String DELETE_ITEM         = "deleteItem";
+    public static final String NEW_FILE            = "newFile";
+    public static final String CREATE_PROJECT      = "createProject";
+    public static final String IMPORT_PROJECT      = "importProject";
+    public static final String CLOSE_ACTIVE_EDITOR = "closeActiveEditor";
+    public static final String SIGNATURE_HELP      = "signatureHelp";
+    public static final String RENAME              = "renameResource";
+    public static final String SHOW_REFERENCE      = "showReference";
+
```

Comment:
Can you move this declarations to the che/ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/action/IdeActions.java ?